EasyPost/Client.cs

```diff
@@ -108,5 +112,35 @@ public Client(ClientConfiguration clientConfiguration)
 
             _dotNetVersion = dotNetVersion;
         }
+
+        /// <summary>
+        /// Return the request time from the client object.
+        /// </summary>
+        public int getConnectionTimeout(){
+            return _defaultConnectTimeoutMilliseconds;
+        }
+
+        /// <summary>
+        /// Set the connection timeout with a time in milliseconds.
+        /// </summary>
+        /// <param name="connectionTimeoutMilliseconds">connection time in milliseconds.</param>
+        public void setConnectionTimeout(int connectionTimeoutMilliseconds){
+            _defaultConnectTimeoutMilliseconds = connectionTimeoutMilliseconds;
```

Comment:
We probably shouldn't set the default connection timeout to something the user sets because it then no longer is the "default", instead (if possible), we should use the user's value only if present and if not, fallback to the default.